bunfig.toml

```
@@ -4,3 +4,4 @@ timeout = 30000
 
 [test.env]
 NODE_ENV = "test"
+BIOME_CONFIG_PATH=biome_madeline.json
```

```suggestion
BIOME_CONFIG_PATH="biome_madeline.json"
```
Isn't this supposed to be in quotes?